setup.py

```diff
@@ -83,11 +85,10 @@ def _named_temporary_file(directory: Optional[str] = None) -> str:
 
 
 @contextlib.contextmanager
-def _set_manifest_path(manifest_dir: str, aggregate: bool = False) -> Generator:
+def _set_manifest_path(manifest_dir: str, aggregate: bool = False, mapping: dict = _PACKAGE_MAPPING) -> Generator:
```

Comment:
Why make this an argument, where is this used?

Reply:
mapping was used so this is makes just cleaner
note that I try tokeep the changes minimal to simplify trace/review

Reply:
Let's keep it as it is cleaner way then have constants inside function...
Also typing shall be another PR as it is unrelated to this change

Reply:
This PR is adding this type annotation. I'm just saying it's bot correct. `Dict` != `dict`